com.github.vikdevelop.photopea_app.yml

```
@@ -0,0 +1,29 @@
+app-id: com.github.vikdevelop.photopea_app
+runtime: org.gnome.Platform
+runtime-version: '40'
+sdk: org.gnome.Sdk
+command: photopea_app
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --share=network
+  - --filesystem=xdg-pictures
+  - --filesystem=xdg-download
+modules:
+  - shared-modules/dbus-glib/dbus-glib-0.110.json
+  - name: photopea_app
+    buildsystem: simple
+    build-commands:
+    - c++ photopea.cpp `pkg-config --cflags --libs gtk+-3.0 webkit2gtk-4.0` -o /app/bin/photopea_app
+    - install -Dm644 -t /app/share/icons/hicolor/128x128/apps/ data/icons/com.github.vikdevelop.photopea_app.png
+    - install -Dm644 -t /app/share/metainfo com.github.vikdevelop.photopea_app.metainfo.xml
+sources:
```

```suggestion
    sources:
```

indentation is wrong breaking the manifest